docs/cudf/source/user_guide/pandas-comparison.md

```diff
@@ -82,9 +82,16 @@ using `.from_arrow()` or `.from_pandas()`.
 
 ## Result ordering
 
-By default, `join` (or `merge`) and `groupby` operations in cuDF
-do *not* guarantee output ordering.
-Compare the results obtained from Pandas and cuDF below:
+In Pandas, `join` and `groupby` operations provide certain guarantees
+about the order of rows in the result returned.  In Pandas `join`, the
+order of join keys are either preserved or sorted lexicographically by
```

Comment:
```suggestion
order of join keys is either preserved or sorted lexicographically by
```
(subject of the sentence is "order" not "join keys")

But I think I don't quite understand the explanation of this restriction (how can ordering be either preserved or sorted by default for a join key?).

Reply:
OK, let us therefore write something like:

> In a Pandas `join`, the order of join keys is (depending on the particular style of join being performed) either preserved or sorted lexicographically.

Reply:
Added this.